scripts/generate-git-snapshot

```diff
@@ -234,7 +234,7 @@ identify_latest_change() {
   fi
 
   echo "Latest tag [${last_tag:-}] / merge [${last_merge:-}] seems to be $since"
-  ${GBP_DCH} -s "${since}" $DCH_COMBINED_OPTS
+  ${GBP_DCH} -s "${since}" $DCH_COMBINED_OPTS $DCH_CHANGELOG_FILE
```

Comment:
Thanks, I'd squash the commits and slightly adjust the commit message (keeping your name though to give you credits), any objections against that?